src/sentry/tagstore/legacy/backend.py

```diff
@@ -262,56 +263,96 @@ def get_tag_key(self, project_id, environment_id, key, status=TagKeyStatus.VISIB
             qs = qs.filter(status=status)
 
         try:
-            return qs.get()
-        except TagKey.DoesNotExist:
+            instance = qs.get()
+        except models.TagKey.DoesNotExist:
             raise TagKeyNotFound
 
+        return TagKey(
+            key=instance.key,
+            values_seen=instance.values_seen,
+            status=instance.status,
+        )
```

Comment:
These transformations are repeated a lot, I can factor them into methods if anybody has a strong opinion about it.

Reply:
Definitely not a strong opinion, but given the attributes all have the same names as the model objects, is there a simple way to do some kind of generic copy-constructor type thing for these objects like `__init__(**instance.__dict__):` ? Or maybe thats too clever, idk.